core/dbt/parser/unit_tests.py

```diff
@@ -219,6 +222,33 @@ def __init__(self, schema_parser: SchemaParser, yaml: YamlBlock) -> None:
         self.schema_parser = schema_parser
         self.yaml = yaml
 
+    def _load_rows_from_seed(self, ref: Dict[str, str]) -> List[Dict[str, Any]]:
```

Comment:
at this point the structure of `ref` feels fairly specific - the caller will have to do some preprocessing to make use of the method correctly. I would just roll the `py_extract_from_source` call directly in this method and accept the raw input `ref_str` instead.